<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>367</width>
    <height>346</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(63, 64, 85);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>241</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="labelUsuario">
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>60</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color: #ffffff</string>
         </property>
         <property name="text">
          <string>Usuario: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineUser">
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #ffffff;border-radius: 150px;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="labelSenha">
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>60</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color: #ffffff</string>
         </property>
         <property name="text">
          <string>Senha:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="linePasswd">
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #ffffff;border-radius: 150px;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btnLogin">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>190</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(178, 221, 231);</string>
    </property>
    <property name="text">
     <string>LOGIN</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelNotificacao">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>160</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 85, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="labelConfirma">
       <property name="styleSheet">
        <string notr="true">color: #ffffff</string>
       </property>
       <property name="text">
        <string>Confirmar senha:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineConfirma">
       <property name="maximumSize">
        <size>
         <width>180</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #ffffff</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btnNovoUsuario">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>151</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #ffa399;border: none;</string>
    </property>
    <property name="text">
     <string>Ainda nao tem cadastro?</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>367</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
